integration/pom.xml

```diff
@@ -788,6 +794,7 @@
                 <module>tests-common</module>
                 <module>mediation-tests</module>
 <!--                <module>dataservice-hosting-tests</module>-->
+                <module>cli-tests</module>
```

Comment:
We can't have this in the default profile

Reply:
Added module to the separate profile.